scripts/quickstart_run.sh

```diff
@@ -55,30 +53,24 @@ if [ $? != 0 ]; then
 fi
 set -e
 
-# create argo namespace if it doesn't exist
 set +e
-kubectl get namespace argo &> /dev/null
-if [ $? != 0 ]; then
-  echo "creating argo namespace"
-  kubectl create namespace argo
-fi
+docker build --pull --rm -f "Dockerfile" -t argocloudops:latest "."
+kubectl apply -f ./scripts/quickstart_manifest.yaml
+# Sleeping after applying manifest so pods have time to start
+sleep 60
```

Comment:
Instead of just sleeping and having a possible race condition on a slow machine,  can we run a loop and check a status etc?